### PR TITLE
Better Labels in About Activity ( as seen in #1846)

### LINKED ISSUE #1846: Improve the labels and summary texts in About
#### Software and hardware versions 
Collect v1.12.2

#### Problem description
The labels and summary texts for the About screen can be improved

Suggestion from @yanokwa  https://github.com/opendatakit/collect/pull/1802#pullrequestreview-95583399

```
 - Share ODK Collect
 - Are your colleagues still collecting data on paper? Share ODK Collect with them.

 - Leave a Play Store review
 - Your (hopefully positive) review increases the visibility of the App in the Play Store.

 - Open source libraries/licenses
 - We stand on the shoulders of giants!
```

### PR DESCRIPTION
**Pull request by raks097**
Closes #1846

I have edited the label and summaries directly from strings.xml

What has been done to verify that this works as intended?
I have run this on my physical Android device and works as intended.
Device Specs:
Sony Xperia C4 Dual E5363 ( Android 6.0)

Why is this the best possible solution? Were any other approaches considered?
The titles are crisp and short while the summaries have more details regarding the options.
This makes it more intuitive and engaging to the user.

Are there any risks to merging this code? If so, what are they?
The strings.xml file for other languages need to be updated as well to maintain a state of consistency
(Help from someone with linguistic proficiency is required)

Do we need any specific form for testing your changes? If so, please attach one.

No testing needed

### CODE REVIEW

**Comment by lognaturel** [comment]
Please use the text from the issue. Please make sure there are spaces after commas in parameter lists.

**Comment by raks097** [comment]
@lognaturel 
Sorry, I assumed they were only suggestions.
I have committed the required changes to the PR.

**Review by lognaturel — APPROVED** [review]

**Comment by lognaturel** [comment]
Thank you, @raks097! 

I thought I had described why we were waiting on merging this but realized I forgot to. When changing strings, we need to leave enough time for translators before a release. Because we had v1.13.0 coming out, it was better to wait until after to merge. Now translators at transifex.com/opendatakit/ will get the new strings and be able to translate them before v1.14.0 comes out.

**Pull request merged**
